charts/exivity/templates/proximity/api.deployment.yaml

```
@@ -21,6 +21,9 @@ spec:
         checksum/{{- include "exivity.fullname" $ -}}-licence-pub: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
         checksum/{{- include "exivity.fullname" $ -}}-licence-key: {{ include (print $.Template.BasePath "/proximity/api.secret.yaml") . | sha256sum }}
         checksum/{{- include "exivity.fullname" $ -}}-lock: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
+        checksum/{{- include "exivity.fullname" $ -}}-proximity-api-env: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
+        checksum/{{- include "exivity.fullname" $ -}}-app-key: {{ include (print $.Template.BasePath "/proximity/api.secret.yaml") . | sha256sum }}
+        checksum/{{- include "exivity.fullname" $ -}}-jwt-secret: {{ include (print $.Template.BasePath "/proximity/api.secret.yaml") . | sha256sum }}
```

Checksum of the secret is already done from `licence-key`

```suggestion
        checksum/{{- include "exivity.fullname" $ -}}-licence-pub: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
        checksum/{{- include "exivity.fullname" $ -}}-secret: {{ include (print $.Template.BasePath "/proximity/api.secret.yaml") . | sha256sum }}
        checksum/{{- include "exivity.fullname" $ -}}-lock: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
        checksum/{{- include "exivity.fullname" $ -}}-proximity-api-env: {{ include (print $.Template.BasePath "/proximity/api.configmap.yaml") . | sha256sum }}
        checksum/{{- include "exivity.fullname" $ -}}-app-key: {{ include (print $.Template.BasePath "/proximity/api.secret.yaml") . | sha256sum }}
```